docs/guides/postgres/initialization/script_source.md

```diff
@@ -14,12 +14,13 @@ section_menu_id: guides
 
 # Initialize PostgreSQL with Script
 
-KubeDB supports PostgreSQL database initialization.
+KubeDB supports PostgreSQL database initialization. This tutorial will show you how to use KubeDB to initialize a PostgreSQL database from script.
 
 ## Before You Begin
 
-At first, you need to have a Kubernetes cluster, and the kubectl command-line tool must be configured to communicate with your cluster.
-If you do not already have a cluster, you can create one by using [minikube](https://github.com/kubernetes/minikube).
+At first, you need to have a Kubernetes cluster, and the kubectl command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [minikube](https://github.com/kubernetes/minikube).
+
+#### Install KubeDB:
```

Comment:
Use sequentially incrementing Heading. 

However, Let's not break `Before You Begin`, specially not for `Install KubeDB`. Use `List` instead of Heading/Section for multiple steps on `Before You Begin`.

Reply:
fixed